modules/unitbar.lua

```diff
@@ -63,6 +63,10 @@ function GladiusEx:NewUnitBarModule(name, defaults_arena, defaults_party)
 			self.frame[unit].statusbar:SetStatusBarColor(color.r, color.g, color.b, color.a or 1)
 			self.frame[unit].icon:SetTexture([[Interface\Glues\CharacterCreate\UI-CharacterCreate-Classes]])
 
+			-- V: wtf is "CLASS_BUTTONS"?
+			if not CLASS_BUTTONS then

```

Comment:
IIRC, **Interface\Glues\CharacterCreate\UI-CharacterCreate-Classes** was a texture used to contain all the class icons in a single texture. CLASS_BUTTONS used to be a global from the wow UI that had the texture coordinates for each class. I don't know if it has been renamed to something else or entirely removed.